starter-code/views/employee/add.hbs

```diff
@@ -0,0 +1,8 @@
+<form action="/auth/login" method="POST">
```

Comment:
The action for this form should be "/employee/add", you also need to add an input to define the role of the user.